clients/fides-js/src/components/ExperienceDescription.tsx

```diff
@@ -5,14 +5,29 @@ const TEXT_TO_LINK = "vendors page.";
 const ExperienceDescription = ({
   description,
   onVendorPageClick,
+  allowHTMLDescription = false,
 }: {
   description: string | undefined;
   onVendorPageClick?: () => void;
+  allowHTMLDescription?: boolean | null;
 }) => {
   if (!description) {
     return null;
   }
 
+  // If allowHTMLDescription is true, render rich HTML content
+  // NOTE: We sanitize these descriptions server-side when configuring the
+  // PrivacyExperience, so it's safe to trust these
+  if (allowHTMLDescription) {
+    return (
+      <div
+        className="fides-html-description"
```

Comment:
This class might be more useful in the future; right now I use it to set `white-space: normal` since we're rendering HTML and therefore no longer want to treat whitespace (specifically newline characters!) as semantic in the same way we do for plaintext.

That CSS is in `fides.css` though, you'll see it later 👍